test/integration/signup_logon_test.js

```diff
@@ -100,6 +96,14 @@ casper.test.begin('Sign up and log on', 6, function(test) {
     test.assert(text.indexOf('unlisted, unguessable') > -1, 'there is a valid show-blocks URL');
   });
 
+  casper.thenOpen(host + '/logout', function() {
```

Comment:
The logout and login stanzas appear to be common between both tests, and will probably be used in a lot of the tests. Can you factor them out into a shared function?